controllers/cluster_controller_test.go

```diff
@@ -1100,6 +1096,8 @@ func TestClusterReconcilerReconcileSelfManagedClusterRegAuthFailNoSecret(t *test
 			ReconciledGeneration: 1,
 		},
 	}
+	eksaRelease := test.EKSARelease()
+	bundles := createBundle()
```

Comment:
i recommend moving the instantiation of these variables closer to where they are used.